plugins/aws/fix_plugin_aws/resource/rds.py

```diff
@@ -342,7 +342,43 @@ class AwsRdsInstance(RdsTaggable, AwsResource, BaseDatabase):
         "ctime": S("InstanceCreateTime"),
         "arn": S("DBInstanceArn"),
         "db_type": S("Engine"),
-        "db_status": S("DBInstanceStatus"),
+        "db_status": S("DBInstanceStatus")
+        >> MapEnum(
+            {
+                "available": DatabaseInstanceStatus.AVAILABLE,
+                "backing-up": DatabaseInstanceStatus.BUSY,
+                "configuring-enhanced-monitoring": DatabaseInstanceStatus.BUSY,
+                "configuring-iam-database-auth": DatabaseInstanceStatus.BUSY,
+                "configuring-log-exports": DatabaseInstanceStatus.BUSY,
+                "converting-to-vpc": DatabaseInstanceStatus.BUSY,
+                "creating": DatabaseInstanceStatus.BUSY,
+                "delete-precheck": DatabaseInstanceStatus.BUSY,
+                "deleting": DatabaseInstanceStatus.TERMINATED,
+                "failed": DatabaseInstanceStatus.FAILED,
+                "inaccessible-encryption-credentials": DatabaseInstanceStatus.FAILED,
+                "inaccessible-encryption-credentials-recoverable": DatabaseInstanceStatus.BUSY,
+                "incompatible-network": DatabaseInstanceStatus.FAILED,
+                "incompatible-option-group": DatabaseInstanceStatus.FAILED,
+                "incompatible-parameters": DatabaseInstanceStatus.FAILED,
+                "incompatible-restore": DatabaseInstanceStatus.FAILED,
+                "insufficient-capacity": DatabaseInstanceStatus.FAILED,
+                "maintenance": DatabaseInstanceStatus.BUSY,
+                "modifying": DatabaseInstanceStatus.BUSY,
+                "moving-to-vpc": DatabaseInstanceStatus.BUSY,
+                "rebooting": DatabaseInstanceStatus.BUSY,
+                "resetting-master-credentials": DatabaseInstanceStatus.BUSY,
+                "renaming": DatabaseInstanceStatus.BUSY,
+                "restore-error": DatabaseInstanceStatus.FAILED,
+                "starting": DatabaseInstanceStatus.BUSY,
+                "stopped": DatabaseInstanceStatus.STOPPED,
+                "stopping": DatabaseInstanceStatus.BUSY,
+                "storage-config-upgrade": DatabaseInstanceStatus.BUSY,
+                "storage-full": DatabaseInstanceStatus.BUSY,
```

Comment:
Isn't storage full more of a failed state? It means I can't write any more data to my DB.

Reply:
I agree, `"storage-full"` rather FAILED than BUSY. But FAILED status also incorrect, because `storage full` is just indicator that db can't write any additional data